2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_setup.py:_flush():68] Configure stats pid to 1840445
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_setup.py:_flush():68] Loading settings from /cluster/raid/home/zhivar.sourati/.config/wandb/settings
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_setup.py:_flush():68] Loading settings from /cluster/raid/home/zhivar.sourati/logical-fallacy-identification/AMR_parsers_and_graphs/wandb/settings
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'AMR_parsers_and_graphs/gcn.py', 'program': '/cluster/raid/home/zhivar.sourati/logical-fallacy-identification/AMR_parsers_and_graphs/gcn.py'}
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:_log_setup():476] Logging user logs to /cluster/raid/home/zhivar.sourati/logical-fallacy-identification/AMR_parsers_and_graphs/wandb/run-20220920_235718-3i45ooxo/logs/debug.log
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:_log_setup():477] Logging internal logs to /cluster/raid/home/zhivar.sourati/logical-fallacy-identification/AMR_parsers_and_graphs/wandb/run-20220920_235718-3i45ooxo/logs/debug-internal.log
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:init():516] calling init triggers
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:init():569] starting backend
2022-09-20 23:57:18,309 INFO    MainThread:1840445 [wandb_init.py:init():573] setting up manager
2022-09-20 23:57:18,314 INFO    MainThread:1840445 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-20 23:57:18,320 INFO    MainThread:1840445 [wandb_init.py:init():580] backend started and connected
2022-09-20 23:57:18,324 INFO    MainThread:1840445 [wandb_init.py:init():658] updated telemetry
2022-09-20 23:57:18,335 INFO    MainThread:1840445 [wandb_init.py:init():692] communicating run to backend with 30 second timeout
2022-09-20 23:57:18,704 INFO    MainThread:1840445 [wandb_run.py:_on_init():1974] communicating current version
2022-09-20 23:57:18,779 INFO    MainThread:1840445 [wandb_run.py:_on_init():1978] got version response 
2022-09-20 23:57:18,779 INFO    MainThread:1840445 [wandb_init.py:init():727] starting run threads in backend
2022-09-20 23:57:23,795 INFO    MainThread:1840445 [wandb_run.py:_console_start():1954] atexit reg
2022-09-20 23:57:23,796 INFO    MainThread:1840445 [wandb_run.py:_redirect():1812] redirect: SettingsConsole.WRAP_RAW
2022-09-20 23:57:23,797 INFO    MainThread:1840445 [wandb_run.py:_redirect():1877] Wrapping output streams.
2022-09-20 23:57:23,797 INFO    MainThread:1840445 [wandb_run.py:_redirect():1899] Redirects installed.
2022-09-20 23:57:23,797 INFO    MainThread:1840445 [wandb_init.py:init():764] run started, returning control to user process
2022-09-20 23:57:43,540 WARNING MsgRouterThr:1840445 [router.py:message_loop():77] message_loop has been closed
